src/main/java/pl/schoolmanager/entity/School.java

```diff
@@ -112,5 +118,32 @@ public void setStudent(Set<Student> student) {
 	public String getNameForForm() {
 		return this.name + " (" +this.type +")";
 	}
+
+	public TimeTable getTimeTable() {
+		return timeTable;
+	}
+
+	public void setTimeTable(TimeTable timeTable) {
+		this.timeTable = timeTable;
+	}
+	
+	public List<String> getTimeTableList() {
+		TimeTable tt = this.timeTable;
+		if (tt == null) {
+			tt = new TimeTable();
+		}
+		List<String> timeTableList = new ArrayList<>();
+		LocalTime first = tt.getStart();
+		LocalTime start = first;
+		LocalTime finish = start.plusMinutes(45);
+		timeTableList.add(start + " - " + finish);
+		
+		for (int i = 0; i < 7; i++) {
+			start = finish.plusMinutes(tt.getList().get(i));
+			finish = start.plusMinutes(45);
+			timeTableList.add(start + " - " + finish);
```

Comment:
you can do like this: tt.getBreaks().get(i);

or iterate over map like this:
for(Map.Entry<Integer, Integer> entry : tt.getBreaks().entrySet()) {
    start = finish.plusMinutes(entry.getValue());
    // code
}